web/js/util/wv.util.js

```diff
@@ -807,6 +807,11 @@ wv.util = (function(self) {
     return value;
   };
 
+
+  self.capitalizeFirstLetter = function(string) {
```

Comment:
I don't think we should add this as a utility function. It adds more noise and complexity to this file, and makes the code harder to follow. This is a pretty simple operation. We can just do this in `wv.layers.modal.js`;
```
orbitTitle = _.startCase(layer.daynight)  + "/" + _.startCase(layer.track);
```
That's shorter and more clear anyway.

Reply:
Also, what case is the string in to begin with?

Reply:
Agreed. This makes a lot of sense. I will remove the function and just use it inline since it is so short. <3 lodash

Reply:
It is start case. The Daynight and Track need to start with first letter uppercased. _.startCase should be the correct function to use here. Ben's function above makes sure everything is lowercase first before converting to the start case but I don't think all that is needed.

Reply:
It's already start case to begin with? Why do we need to convert it?

Reply:
Negative. They are lowercase values to begin with. I meant that they need to be start case.